lecli/saved_query_api.py

```diff
@@ -0,0 +1,191 @@
+"""
+Saved Query API module.
+"""
+import click
+import requests
+
+from lecli import api_utils
+from lecli import response_utils
+
+
+def _url():
+    """
+    Get rest query url of account resource id.
+    """
+    return 'https://rest.logentries.com/query/saved_queries'
```

Comment:
we are mocking this url in unit tests as httpretty library has a bug regarding mocking ssl requests: https://github.com/gabrielfalcao/HTTPretty/issues/35

on the other hand, other modules have the same method, with different implementations, depending on the endpoint: https://github.com/rapid7/lecli/blob/master/lecli/team_api.py#L11